City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-8202 group menu translation

# [UHF-8202](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8202)
Sidebar menus on highschool pages show all content on all languages

## What was done
Added language manipulator via eventsubscriber which hides links without correct translation.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-8202_group_menu_translation
  * `make fresh`
* Run `make drush-cr`

## How to test
Select one of the highschool groups, for example etutöölön lukio.
Make the selected group has proper content.
  - content and translations for the content.  
Go to highschool page, (for example /fi/kasvatus-ja-koulutus/etu-toolon-lukio/etu-toolon-lukion-yhteystiedot)
Check the sidebar menu - it should show content only with currently selected language
Try editing the group menu. Editing should work without problems. 



[UHF-8202]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8202?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
